airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -369,7 +375,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
             json_schema["properties"]["properties"] = {"type": "object", "properties": self.properties}
         return json_schema
 
-    @retry_connection_handler(max_tries=2)
+    @retry_connection_handler(max_tries=5)
```

Comment:
@a-rampalli why are we increasing the count of retries here ?

Reply:
Airbyte is retrying this error 5 times and in general all retries we are doing are either 3 or 5. Just this nothing specific

Reply:
Resolving as PR merge not allowed with open comments